### ISSUE TITLE
Parallel layer reflectance

### ISSUE THREAD

**Issue by hugadams** [hugadams/fibersim#3]
It would be nice to have a parallel layer that can compute the reflectance from a fiber with different species on (eg np's, clusters, super clusters).  

First, I should do better sims (see already done one in gist) and figure out how to get better estimations of clusters (MG mixing variable K?) on the surface.   Then I can look at the reflectance and sensitivity of these one-by-one.  Experience says that clusters fuck shit up, as fibers with lots of clusters early don't work well.  Makes sense since a cluster's singal should dominate any change from a protein.

After doing these preliminary results, think of about if it's worthwhile trying to develop a layer that would compute these in parallel.  In reality, maybe it's not that the layer should have three paralell computations, but really three paralell stacks should be computed instead.  This would make each computation possible with the current system.
